src/html_support_files/odoc.css

```diff
@@ -1077,7 +1077,7 @@ td.def-doc *:first-child {
 }
 
 .odoc-toc ul li {
-  margin: 0;
+  margin: 8px 0;
```

Comment:
This creates gaps in the line formed by each item's left border. Changing the `padding` instead would be better.

Reply:
Thanks! It does look better that way.